bitrise.yml

```diff
@@ -138,6 +138,25 @@ workflows:
           inputs:
             - artifact_sources: build-and-test.test-build
 
+  # Helper step to start snowplow
+  _start_snowplow:
+    steps:
+        - script@1:
+           inputs:
+             - content: |
+                #!/usr/bin/env bash
+
+                # fail if any commands fails
+                set -e
+                
+                ## Usually Pocket runs snowplow via our docker image, but Bitrise does not support docker in Mac environments so we use their built jar.
+                ## It uses the values in MICRO_IGLU_REGISTRY_URL and MICRO_IGLU_API_KEY
```

Comment:
Note this. I think @brahmini also uses the raw jar in her test suite, but calling this out cause this is different then some other teams, but a side of effect of Mac CI 😬

Reply:
Catching up late here, but, yes we use java jar on the web :)